lib/tty/prompt/list.rb

```diff
@@ -13,10 +15,15 @@ class Prompt
     class List
       include Symbols
 
-      HELP = '(Use arrow%s keys, press Enter to select)'
+      HELP = '(Use arrow%s keys, press Enter to select%s)'
 
       PAGE_HELP = '(Move up or down to reveal more choices)'
 
+      KEY_CANC = "\e[3~".freeze
```

Comment:
Ideally, any low level terminal codes should be handled by `tty-reader` and we should rely on its api. Partly, because this library needs to work on different systems including Windows, which it is currently failing on, and keys have very different codes there. I've noticed your ticket in `tty-reader` repository, thank you!